test/cli/autocomplete_test.py

```diff
@@ -8,7 +8,7 @@
     "incomplete,expected",
     [
         ["an", ["ansi"]],
-        ["s", ["snowflake", "spark3", "sqlite"]],
+        ["s", ["snowflake", "sparksql", "sqlite"]],
```

Comment:
There are a few more references in comments on lines 112, 123, and 134

Reply:
Found the remaining reference in `L026.py` and fixed with new commit, but I couldn't find the ones you're referring to here.

Reply:
Sorry this was in rule L057.py (was the file above this one in the review tab, and clearly I clicked the wrong one!)

Reply:
Fixed! Sorry I missed these references in the rule files!